htdocs/display.php

```diff
@@ -126,7 +126,7 @@
             }
 
             # Lock
-            $pwdLockout = strtolower($ppolicy_entry[0]['pwdlockout'][0]) == "true" ? true : false;
+            $pwdLockout = strtolower(isset($ppolicy_entry[0]['pwdlockout'][0])) == "true" ? true : false;
```

Comment:
No.

The value of `$ppolicy_entry[0]['pwdlockout'][0]` is TRUE (or FALSE, or nonexistent)

This value is the one of pwdLockout attribute in LDAP directory

Also, please don't mix multiple features/bug fixes in the same commit or pull request.